Periodic_4_hyperbolic_triangulation_2/test/Periodic_4_hyperbolic_triangulation_2/test_p4ht2_matrix.cpp

```diff
@@ -0,0 +1,47 @@
+#include <CGAL/CORE_Expr.h>
+#include <CGAL/Point_2.h>
```

Comment:
Do not include Point_2.h and the like as including a kernel does this.

Reply:
Done.